.gitignore

```diff
@@ -1,2 +1,7 @@
 benchmark_data/*
+prepared_data/*
 *.swp
+__pycache__
+.env/*
+.venv/*
```

Comment:
Several of these seem to be personal folders that others will not have in their local setup, so I think it would be better to remove them.